packages/main/src/webComponents/ResponsivePopover/ResponsivePopover.stories.mdx

```diff
@@ -116,6 +117,37 @@ const MyComponentWithPopover = () => {
 };
 ```
 
+**Opening a `ResponsivePopover` by reference and not by `id`**
+
+The web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```

Comment:
```suggestion
This web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```

Reply:
You're right, this is better. Done :)